src/relay/backend/contrib/dnnl/codegen.cc

```diff
@@ -53,163 +141,186 @@ class CodegenDNNL : public ExprVisitor, public CodegenCBase {
   }
 
   void VisitExpr_(const TupleGetItemNode* op) final {
-    // Do nothing
+    VisitExpr(op->tuple);
```

Comment:
I wondered why this change is on my diff, but it seems TupleGetItemNode handling was reverted in #5277